src/test/java/core/basesyntax/storage/DataBaseImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.storage;
+
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DataBaseImplTest {
+    private Storage storage;
+
+    @Before
+    public void setUp() {
+        storage = new StorageImpl();
+    }
+
+    @After
+    public void tearDown() {
+        storage.clearStorage();
+    }
+
+    @Test
+    public void addFruits_Ok() {
+        storage.add("apple", 100);
+        Set<Map.Entry<String, Integer>> actualResult = storage.getEntrySet();
+        Set<Map.Entry<String, Integer>> expectedResult = Map.of("apple", 100).entrySet();
+        Assert.assertEquals(expectedResult, actualResult);
+        storage.remove("apple", 100);
```

Comment:
there is annotation which allows you to do some actions (for example, clear storage) after every test case